LEAF_Nexus/js/employeeSelector.js

```diff
@@ -67,7 +67,7 @@ employeeSelector.prototype.showBusy = function() {
 
 employeeSelector.prototype.select = function(id) {
 	this.selection = id;
-
+	if(typeof event.key !== 'undefined' && event.key !== 'Enter') return;
```

Comment:
Lets get into the mindset of always "cleaning" data before doing comparisons.  In this case, make event.key all lower case and compare to "enter".